lib/api/core/statistics.js

```diff
@@ -174,15 +174,15 @@ class StatisticsController {
 
 
     if (request && request.input.args && request.input.args.startTime) {
-      startTime = new Date(request.input.args.startTime).getTime();
+      startTime = isNaN(request.input.args.startTime) ? new Date(request.input.args.startTime).getTime() : request.input.args.startTime;
     }
 
     if (request && request.input.args && request.input.args.stopTime) {
-      stopTime = new Date(request.input.args.stopTime).getTime();
+      stopTime = isNaN(request.input.args.stopTime) ? new Date(request.input.args.stopTime).getTime() : request.input.args.stopTime;
     }
 
     if ((startTime !== undefined && isNaN(startTime)) || (stopTime !== undefined && isNaN(stopTime))) {
-      return Bluebird.reject(new BadRequestError('Invalid time value'));
+      return Bluebird.reject(new BadRequestError('Invalid time value - StartTime: '+request.input.args.startTime+' => '+startTime+' -- StopTime:'+stopTime));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this concatenation to the use of a template. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3512)